modules/Virtualizer_module/src/VirtualizerModule.cpp

```diff
@@ -206,9 +206,10 @@ bool VirtualizerModule::updateModule()
 
     double tmpSpeedDirection = (double)(m_cvirtDeviceID->GetMovementDirection());
     double speedDirection = 1.0; // set the speed direction to forward by default.
+    
     if (std::abs(tmpSpeedDirection) < 0.01)
         speedDirection = 1.0;
-    else if (std::abs(tmpSpeedDirection - 1) < 0.01)
+    else if (std::abs(tmpSpeedDirection + 1) < 0.01)
```

Comment:
Why this change? And what is this ``1``?

Reply:
Basically, in the first version of the sdk, if the person was walking backward in the sdk the output value from the sdk indicating the direction of the motion was `+1`. In the new version of the SDK, this value is `-1` (which makes more sense actually)!
I'll add a comment on top of it explaining this value.

Reply:
Ok, then if the value is negative, isn't it sufficient to check directly the sign of ``tmpSpeedDirection``?

Reply:
No, since the sdk method is returning a float variable, I wanted to be sure if the value we are receiving is correct. For example, in case for some reason the variable is not set or initialized correctly.

Reply:
I have added the commnet.

Reply:
> No, since the sdk method is returning a float variable, I wanted to be sure if the value we are receiving is correct. For example, in case for some reason the variable is not set or initialized correctly.

To be honest, this is not really clear to me. Why is it different from doing
```c++
if (tmpSpeedDirection < -0.99)
```
? I usually prefer having magic numbers around. Even if I get the meaning, the reason of having that ``+1`` is not immediately clear to me.

Reply:
there is the difference, if you check there is the distance from `-1` so it clearly different from `if (tmpSpeedDirection < -0.99)`.With the comment of the line, should be cristal clear now.

Reply:
Yeah I got meaning, but I am not understanding why you are checking it in this way. In my opinion that condition can be simplified to be more direct, but anyway it is not blocking.

Reply:
Which part of it you did not understand?!

Reply:
We quickly discussed F2F. What I did not understand was the ``abs`` was used as a distance metric from the point ``-1``. The condition is valid only if the  ``tmpSpeedDirection`` is in the range ``[-1.01, -0.99]``.